cmd/esc/cli/cli_test.go

```diff
@@ -968,6 +1000,16 @@ func (c *testPulumiClient) ListEnvironmentRevisionTags(
 	})), nil
 }
 
+// unused
```

Comment:
Why is this unused?

Reply:
The test client doesn't use it. I'm still adding some tests so this may or may not change